### DISCUSSION TITLE
Training Log 2022-07-03

### DISCUSSION THREAD

**Discussion by zh-zheng** [OpenBMB/CPM-Live#52] [Training Logs 训练日志]
# CPM-Live Training Log (July, 3)

Time: July, 3 2022 16:00

Recorder: @zh-zheng

## Loss
- Begin: 2.60
- End: 2.63
	
## Completed Data
- $\approx$ 424.96GB

## Average Grad Norm
- 0.92

## Progress
- 45.63%

## Comment

Even though the training loss has been increasing slightly for two consecutive days, we still think it's not a big deal. We will continue to monitor the loss to determine if there are problems with our model.
